bin/kafka-server-start.sh

```diff
@@ -19,6 +19,14 @@ then
 	echo "USAGE: $0 [-daemon] server.properties [--override property=value]*"
 	exit 1
 fi
+
+for arg in "$@"; do
+    if [[ "$arg" == *.properties ]]; then
+        [[ "$arg" == *"config/kraft"* ]] && echo "Warning: The 'config/kraft' path has been removed. Now all configuration files are in 'config'."
```

Comment:
Should the pattern be `*"config/kraft/"*`? Otherwise you get the message printed if your config file is called `kraft.properties` for example.
If so let's do it in `bin/kafka-storage.sh` too.

I'm actually wondering if we need this warning at all. Users could get this message if their configuration files are in custom paths. What do the other committers think?

Reply:
I agree on the trailing slash pattern. This would better distinguish directories from files like `kraft*.properties.`
I will update it soon.

As for the warning, I also don't think it is necessary since the user might put their config files wherever they want. But I'm open for this solution. If we eventually want to keep this warning, we could make it more informative, as mentioned before, it might catch the user custom path.

Reply:
@chia7712 @ijuma @jsancio Should we have these warnings? WDYT?

Reply:
Printing warnings might be awkward in this scenario, as it can be difficult to determine whether the configuration file originates from the 'removed' folder. Users might continue to utilize the path `config/kraft`.

Maybe we can just remove the `config/kraft`.

Reply:
It seems that most people consider not leaving this warning (as it's not 100% accurate), so I have removed them.